phoenix-core/src/main/java/org/apache/phoenix/mapreduce/util/IndexColumnNames.java

```diff
@@ -79,6 +85,8 @@ public IndexColumnNames(final PTable pdataTable, final PTable pindexTable) {
                 dataColSqlTypeNames.add(getDataTypeString(dPkCol));
                 indexPkColNames.add(indexColumnName);
                 indexColSqlTypeNames.add(getDataTypeString(indexCol));
+                indexColSqlTypeNamesForSkipScan.put(indexColumnName, getDataTypeString(dPkCol));
```

Comment:
Is using skipscan for indexscrutiny optional? If not, can the above two lists be merged into?

Reply:
It is not optional but I need these both.